### PR TITLE
Add README files for various extensions

### PR DESCRIPTION
**Pull request by jopejoe1**
Going to add more readme files. Just putting it here so people can review the ones i already finished.

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [ ] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: src/all/lanraragi/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# LANraragi
+
+Table of Content
+- [FAQ](#FAQ)
+- [Guides](#Guides)
```
Comment (ghostbear) [inline src/all/lanraragi/README.md:5]:
```suggestion
- [FAQ](#FAQ)
  - [Why do I see no manga?](#why-do-i-see-no-manga)
  - [Where can I get more information about LANraragi?](#where-can-i-get-more-information-about-lanraragi)
  - [The LANraragi extension stopped working?](#the-lanraragi-extension-stopped-working)
  - [Can I add more than one LANraragi server or user?](#can-i-add-more-than-one-lanraragi-server-or-user)
  - [Can I test the LANraragi extension before setting up my own server?](#can-i-test-the-lanraragi-extension-before-setting-up-my-own-server)
- [Guides](#Guides)
  - [How do I add my LANraragi server to Tachiyomi?](#how-do-i-add-my-lanraragi-server-to-tachiyomi)
```

**Review by ghostbear — COMMENTED** [review]
You could add the titles to the Table of Content that way it is easier to overview what the README contains

**Review by ThePromidius — COMMENTED** [review]
Kavita has its own page for Tachiyomi in its wiki on how to use and how to install if you want to add it in its guide section

https://wiki.kavitareader.com/en/guides/plugins-and-3rd-party-scripts/tachiyomi

**Comment by Riztard** [comment]
btw why bother adding old version changelog?

**Comment by jopejoe1** [comment]
> btw why bother adding old version changelog?

Looks nicer

**Pull request merged**